luigi/parameter.py

```diff
@@ -223,6 +227,23 @@ def parse(self, x):
         """
         return x  # default impl
 
+    def parse_list(self, xs):
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/) seems to agree with you:

> Public attributes are those that you expect unrelated clients of your class to use, with your commitment to avoid backward incompatible changes. Non-public attributes are those that are not intended to be used by third parties; you make no guarantees that non-public attributes won't change or even be removed.